lib/msf/core/modules/metadata/store.rb

```diff
@@ -22,6 +22,13 @@ def init_store
     load_metadata
   end
 
+  def get_user_store
```

Comment:
Sorry if I'm missing it; where is this helper being used 👀

Reply:
That was in place in the old version that relied on the store. I'll revert the chance since it's no longer necessary.